src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuiteMgt.Codeunit.al

```diff
@@ -6,6 +6,7 @@
 namespace System.TestTools.AITestToolkit;
 
 using System.Reflection;
+using System.Utilities;
```

Comment:
Please sort usings alphabetically.

Reply:
It was auto added when accepting the suggestions, good feedback for AL extension.
I will update this.